docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -116,6 +116,26 @@ Custom yaml settings are silently ignored by {fleet}.
 Settings with input blocks, such as Max agents are still effective.
 ====
 
+[[known-issue-issue-2249-8.6.2]]
```

Comment:
This can be copied to the 8.6.1 and 8.6.0 release notes which are in this same document. This is a bit annoying but let's make sure users are likely to read this regardless of the version of 8.6 they are targeting.

* 8.6.1 notes https://github.com/elastic/ingest-docs/blob/main/docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc#release-notes-8.6.1
* 8.6.0 notes https://github.com/elastic/ingest-docs/blob/main/docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc#release-notes-8.6.0